examples/twopoints.py

```diff
@@ -55,7 +55,7 @@ def callback(self):
                        fill=tuple(from_hsv(self.hue, 1, 1)))
         output = self.img.copy()
         output.thumbnail((28, 14))
-        return np.asarray(output)
+        return np.asarray(output).tolist()
```

Comment:
I'm not sure if we can do this if we don't require numpy. Though this script requires Pillow, so we're relying on implicit dependencies One way or the other. Off top of my head I think we should be able to do

```suggestion
        return output.tolist()
```